hal/src/electron/cellular_hal.cpp

```diff
@@ -7,6 +8,36 @@
 
 static CellularCredentials cellularCredentials;
 
+static HAL_NET_Callbacks netCallbacks = { 0 };
+
```

Comment:
I wonder if the callbacks should take a reserved parameter so we can add more info later?